sanity/doctors/EstateDoctor.js

```diff
@@ -99,6 +120,13 @@ export class EstateDoctor extends Doctor {
 
     return leftIds.every((parcelId, index) => parcelId === rightIds[index])
   }
+
+  isOperatorMismatch(currentOperator, operator) {
+    return (
+      (!operator && !Contract.isEmptyAddress(currentOperator)) ||
```

Comment:
*Question*: wouldn't it be more accurate to do this?:

```suggestion
      (Contract.isEmptyAddress(operator) && !Contract.isEmptyAddress(currentOperator)) ||
```

because the caller could send anything as the first argument, even an empty address